CRM/Activity/Form/Activity.php

```diff
@@ -928,7 +928,11 @@ public function postProcess($params = NULL) {
 
     // store the submitted values in an array
     if (!$params) {
-      $params = $this->controller->exportValues($this->_name);
+      $params = [];
+      $this->exportedValues = $this->controller->exportValues($this->_name);
```

Comment:
So these lines - I propose consolidating them with the `getSubmittedValues()` in the import flow - see https://github.com/civicrm/civicrm-core/pull/25877